app/docs/0.13.x/auth.md

```diff
@@ -4,16 +4,16 @@ title: Authentication Reference
 
 # Authentication Reference
 
-Client access to upstream API services is typically controlled by the application and configuration of 
-Kong [authentication plugins][plugins]. 
+Client access to upstream Services is typically controlled by the application and configuration of
```

Comment:
In this context, "upstream service" refers to the user's backend (API or microservice or [thing]), not to the entity. We only capitalize "Service" when we refer to the Kong entity, and try to differentiate "upstream service/backend/remote server" from "Kong's Service entity".

In this case I think a rephrase is necessary, something like:

Traffic to your upstream services (API or microservice) is typically controlled by the application and configuration of various Kong [authentication plugins][plugins]. Since Kong's Service entity represents a 1-to-1 mapping of your own upstream services, the simplest scenario is to configure authentication plugins on the Services of your choosing.

Feel free to improve though